server/models.py

```diff
@@ -43,6 +43,7 @@ class Election(BaseModel):
     # audit_name must be unique within each Organization
     audit_name = Column(String(200), nullable=False)
     audit_type = Column(Enum(AuditType), nullable=False)
+    ballot_polling_type = Column(Enum(BallotPollingType), nullable=True)
```

Comment:
leave out `nullable=True` since that's the default. that way we only ever see the `nullable` option when it's false